File: src/pe-007.rb
a = [0,2,3,5,7,11,13]
num = 15

def check_prime num
	for i in 2...num
		if num%i == 0
			return false
		end
	end	
	return true
end

while a.count <= 10001
	if check_prime(num)
		a.push(num)
		num+=2
	else
		num+=2
	end
end

puts a.last

File: project-euler/src/js/pe-007.js
function checkPrime(num) {
    for (x=2;x<num/2;x++) {
        if (num%x==0) {
            return false;
        }
    }
    return true;
}

num=1;
i=1;
while (num<10001) {
    i+=2;
    if (checkPrime(i)) {
        num++;
    }
}
console.log(num);
console.log(i);